packages/cactus-common/src/main/typescript/error-utils.ts

```diff
@@ -28,5 +28,16 @@ export function safeStringifyException(error: unknown): string {
     return sanitizeHtml(error.stack || error.message);
   }
 
+  // Axios and possibly other lib errors produce nicer output with toJSON() method.
+  // Use it if available
+  if (
+    error &&
+    typeof error === "object" &&
+    "toJSON" in error &&
+    typeof error.toJSON === "function"
+  ) {
+    error = error.toJSON();
+  }
+
```

Comment:
@outSH I agree with the idea but I have one nit-pick:
If the execution hits this if-condition then `error` will be a JSON string and then we call `safeStringify()` on it which will mangle that by escaping everything in it (all the double quotes, etc.).

Not really a big issue but it's definitely one of those quality of life things that we could improve for people reading the crashlogs.

TLDR: I would refactor it in a way that safeStringify is only called on the else (e.g. if there was no `toJSON()` implementation found on the error)

Reply:
@petermetz Good catch - fixed